sounds/models.py

```diff
@@ -1102,6 +1102,21 @@ def analyze(self, force=False, high_priority=False):
             }), wait_until_complete=False, background=True, priority=gearman.PRIORITY_HIGH if high_priority else None)
             sounds_logger.info("Send sound with id %s to queue 'analyze'" % self.id)
 
+    def analyze_v2(self, force=False, high_priority=False, analyzer="fs-essentia:1"):
```

Comment:
This will need to be reimplemented with celery but it is fine to keep it like that until I get more details about how to do the celery thing